README.md

```diff
@@ -2,11 +2,41 @@
 
 [![CircleCI](https://circleci.com/gh/pkamenarsky/concur-replica.svg?style=svg)](https://circleci.com/gh/pkamenarsky/concur-replica)
 
-`concur-replica` is a [Concur](https://github.com/ajnsit/concur) backend for [Replica](https://github.com/pkamenarsky/replica). As such, you might want to make yourself familiar with both projects first.
+Build interactive web UIs using nothing but Haskell.
+
+No user-written JS. No transpiler.
+
+## How?!
+
+[Replica](https://github.com/pkamenarsky/replica): a remote virtual DOM library. Clients run a fixed JavaScript snippet which opens a connection to your Haskell server. The client uses it to notify the server of browser events, and the server uses it to send DOM updates to be applied by the client.
+
+[Concur](https://github.com/ajnsit/concur): a UI framework that generalizes the Elm architecture. It's based around `Widget`s, sections of the UI that may return a value.
+
+## Upsides and downsides
+
+This strategy is well understood due to its use by [Phoenix LiveView](https://hexdocs.pm/phoenix_live_view) (among other projects).
+
+There are two situations where you should rule `concur-replica` out completely:
+
++ Instant feedback is required
+
++ It needs to work offline
+
+The strategy also has two softer costs:
+
++ Animations costs bandwidth as well as CPU
+
++ It puts a heavier load on your server than a traditional stateless webapp
+
+Additionally, `concur-replica` is alpha software so it currently has additional deficiencies. Check the issue tracker for these, and they'll be fixed as it matures.
+
+Despite these caveats however, `concur-replica` has *incredible* benefits. No JS! No REST API layer-- you can call whatever Haskell functions you want directly from your UI code. `concur` is an awesome way of making UIs. And you have the Haskell classics: STM is absolutely unfair for building collaborative apps, and the type system shines since UIs are so annoying to test.
+
+We may have a real path to making Haskell a best-in-class UI language.
```

Comment:
Sounds a bit unjustified - i.e. it's still impossible to even build a Google Maps clone efficiently (because we'd need to continuously send `onMouseMove` events to the server and wait on the diff), not to mention that with higher pings something like that would be completely unusable.

Maybe better to remove completely?

Reply:
This is a great suggestion.